dataprofiler/profilers/profile_builder.py

```diff
@@ -955,10 +986,6 @@ def _restore_data_labelers(self, data_labeler=None):
                 self.options.set({"data_labeler.is_enabled": False})
                 self.options.set({"data_labeler.data_labeler_object": data_labeler})
 
-            except Exception as e:
-                utils.warn_on_profile("data_labeler", e)
-                self.options.set({"data_labeler.is_enabled": False})
-
```

Comment:
is this change needed as a result of static typing update?

Reply:
I'd rather keep these PRs only static typing and not changing code ... unless its absolutely necessary

Reply:
that second exception i don't think gets hit ever so I think it's fine to remove.

Reply:
I'll make a separate PR for any code changes in the future.

Reply:
perfect, thanks, tony!